### ISSUE TITLE
antiAFK broken

### ISSUE THREAD

**Issue by MrGeorgen** [themoonisacheese/2bored2wait#96]
2b2t kicks you even if antiAFK is turned on. I recently changed the public chat message to pm to rusherBot. To test this takes a lot of time. So if someone has more information regarding 2b2t antiAFK, please let me know

**Comment by dsetareh** [comment]
I just recently had an internet outage at home and 2b2w kept me online for 10min+ as I reconnected. 
This is probably good enough.

If your goal is faster access to 2b2t at all times, I think being in queue 24/7 is better than idling in-game. You always seem to get disconnected from 2b2t after a few hours, not to mention your character could be in dangerous areas while afk. 

Before chunk-caching, I used to script restarting my queue at position 10 if I was not about to play, as not connecting before queue ends was unrecoverable. 

With chunk-caching, 2b2t's antiafk system could work to our favor by seeing you're not online and kicking you so that you'll automatically rejoin queue. I wouldn't want to start playing on a session that's been logged in for 5 hours to get kicked in 30min.
